projects/v3/src/app/components/list-item/list-item.component.ts

```diff
@@ -17,6 +17,7 @@ export class ListItemComponent {
   @Input() leadingIconColor: string;
   @Input() leadingIconPulsing: boolean;
   @Input() leadImage: string;
+  @Input() leadImageClass: string;
```

Comment:
Hi @sasangachathumal this one you may need to add a `?` after the variable name to indicate it's optional.